corehq/apps/styleguide/templates/styleguide/_includes/atoms/accessibility.html

```diff
@@ -98,6 +98,34 @@ <h1 id="atoms-accessibility">Accessibility</h1>
     <li><a href="https://a11yproject.com/">https://a11yproject.com/</a></li>
     <li><a href="http://www.carbondesignsystem.com/guidelines/accessibility">http://www.carbondesignsystem.com/guidelines/accessibility</a></li>
     <li><a href="https://www.w3.org/WAI/tutorials">https://www.w3.org/WAI/tutorials</a></li>
+    <li><a href="https://www.youtube.com/playlist?list=PLNYkxOF6rcICWx0C9LVWWVqvHlYJyqw7g">A11ycast videos</a></li>
+  </ul>
+  </p>
+    <p>
+    In 2020, USH began a web accessibility improvement effort, specifically surrounding screen readers.
+    <a href="https://developers.google.com/web/tools/chrome-devtools/accessibility/reference">This document</a> details
```

Comment:
This url looks like it might not be right.

Reply:
good catch!